game/scripts/vscripts/abilities/oaa_zuus_cloud.lua

```diff
@@ -0,0 +1,114 @@
+zuus_cloud = class( AbilityBaseClass )
+LinkLuaModifier( "modifier_zuus_cloud_oaa", "abilities/oaa_zuus_cloud.lua", LUA_MODIFIER_MOTION_NONE )
+
+function zuus_cloud:OnSpellStart()
+  local caster = self:GetCaster()
+  local hCloud = CreateUnitByName( "npc_dota_zeus_cloud", self:GetCursorPosition(), true, caster, caster, caster:GetTeamNumber() )
+  hCloud:SetOwner( self:GetCaster() )
+  hCloud:SetControllableByPlayer( self:GetCaster():GetPlayerOwnerID(), false )
+  hCloud:AddNewModifier( caster, self, "modifier_zuus_cloud_oaa", { Interval = self:GetSpecialValueFor( "cloud_bolt_interval" ), Radius = self:GetSpecialValueFor( "cloud_radius" ) } )
+  hCloud:AddNewModifier( caster, self, "modifier_kill", { duration = self:GetSpecialValueFor( "cloud_duration" ) } )
+  FindClearSpaceForUnit( hCloud, self:GetCursorPosition(), true )
+end
+
+modifier_zuus_cloud_oaa = class( ModifierBaseClass )
+
+function modifier_zuus_cloud_oaa:IsHidden()
+  return true
+end
+
+function modifier_zuus_cloud_oaa:IsDebuff()
+  return false
+end
+
+function modifier_zuus_cloud_oaa:IsPurgable()
+  return false
+end
+
+function modifier_zuus_cloud_oaa:RemoveOnDeath()
+  return true
+end
+
+function modifier_zuus_cloud_oaa:OnCreated( kv )
+  if IsServer() then
+    self.Interval = kv.Interval
+    self.Radius = kv.Radius
```

Comment:
There isn't really any reason to have these values passed in on creation rather than just being read from the ability here.